docs/marketplace/user/features/202108.0/marketplace-shipment-feature-overview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Marketplace Shipment feature overview
+description: This document contains concept information for the Marketplace Shipment feature.
+template: concept-topic-template
+---
+
+The *Marketplace Shipment* allows splitting the [marketplace order](/docs/marketplace/user/features/{{ page.version }}/marketplace-order-management/marketplace-order-management-overview/marketplace-order-management-overview.html) into different shipments based on merchants who will process it.
+
+A *shipment* is a set of two or more products combined by the same delivery address.
+
+{% info_block infoBox "Info" %}
+
+To learn more about Shipment feature in the SCCOS, see [Shipment feature overview](https://documentation.spryker.com/docs/shipment-feature-overview).
+
+{% endinfo_block %}
+
+
+## Marketplace Shipment on the Storefront
+In the *Address* checkout step, you can define a common delivery address where all the shipments will be delivered.
+Then, in the *Shipment* checkout step, you can see that by default the products are grouped by merchant. For each shipment, you can select a delivery date (optional).
+
+![img](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+Shipment/shipment-to-single-address.png)
+
+Or you can use the **Deliver to multiple addresses** drop-down option that will let you assign a delivery address per cart item.
```

Comment:
here we need to say that by doing that, products from different merchants will not be in the same shipment.

Reply:
added